app/frontend/components/_sign-table.scss

```diff
@@ -0,0 +1,57 @@
+.sign-table-row {
+  border-bottom: 1px #C2CDD1 solid;
```

Comment:
If this is a new colour, add it to the SASS settings file. This allows us to keep a list of the colours we have in use, and allows for reuse.